worlds/apsudoku/docs/setup_en.md

```diff
@@ -13,25 +11,33 @@ Does not need to be added at the start of a seed, as it does not create any slot
 
 ## Installation Procedures
 
-Go to the latest release from the [APSudoku Releases page](https://github.com/EmilyV99/APSudoku/releases). Download and extract the `APSudoku.zip` file.
+Go to the latest release from the [APSudoku Releases page](https://github.com/APSudoku/APSudoku/releases/latest). Download and extract the appropriate file for your platform.
 
 ## Joining a MultiWorld Game
 
-1. Run APSudoku.exe
-2. Under the 'Archipelago' tab at the top-right:
-	- Enter the server url & port number
+1. Run the APSudoku executable.
+2. Under `Settings` &rarr; `Connection` at the top-right:
```

Comment:
Usually you'd say in the top-right
```suggestion
2. Under `Settings` &rarr; `Connection` in the top-right:
```

Reply:
My brain disagrees with you on this one, and a google search gave me results indicating that either are technically correct / it's likely regional differences.

Reply:
https://books.google.com/ngrams/graph?content=at+the+top-left+corner%2C+in+the+top-left+corner&year_start=1800&year_end=2022&corpus=en&smoothing=3
🤷